script/testing/junit/utils.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python3
+
+import argparse
+
+from oltpbench import constants
+
+
+def section_header(title):
+    border = "+++ " + "="*100 + " +++\n"
+    middle = "+++ " + title.center(100) + " +++\n"
+    return "\n\n" + border + middle + border
+# DEF
+
+
+def parse_command_line_args():
```

Comment:
This seems unnecessary to put in here. Why not have it in the script that uses it?

Reply:
Yah, I agree with you. It is not necessary.